src/nvim/api/buffer.c

```diff
@@ -170,20 +192,34 @@ void buffer_set_line_slice(Buffer buffer,
                       ArrayOf(String) replacement,
                       Error *err)
 {
```

Comment:
This and the other functions that take `include_start`/`include_end` should be marked as deprecated. I guess doxygen has a marker for that, but also gcc/clang have an attribute... http://stackoverflow.com/a/12839666/152142